listenbrainz/webserver/__init__.py

```diff
@@ -309,3 +309,7 @@ def _register_blueprints(app):
 
     from listenbrainz.webserver.views.user_timeline_event_api import user_timeline_event_api_bp
     app.register_blueprint(user_timeline_event_api_bp, url_prefix=API_PREFIX)
+
+    if app.config.get("FEATURE_PINNED_REC", False):
```

Comment:
You also need to add the feature flag to this file, https://github.com/metabrainz/listenbrainz-server/blob/master/consul_config.py.ctmpl . I'll update the production config and let you know the exact changes needed to this file.

Reply:
I added FEATURE_PINNED_REC = '''{{template "KEY" "feature_pinned_rec"}}'''  following the pattern from other commits, please let me know if this is sufficient.

Reply:
I updated the consul configs. Can you please merge the latest master into your branch and then change this line to `FEATURE_PINNED_REC = {{template "KEY_JSON" "feature_pinned_rec"}}`.

Reply:
Changed!